### PR TITLE
Home, Agency Index: Use new EN/ES copy with 1 new agency variable

### LINKED ISSUE #1386: Home & Agency Index - Update homepage copy + Create Agency text variables
Update home page copy to be more concise for users.

image

## Acceptance criteria

- [x] Draft English Copy written
- [x] Copy ready for translation vendor 
- [x] Spanish copy added to app
- [x] Execute/ test new process for updating copy and working with content localization vendor

## Definition of "Done"

- [x] New homepage copy published in English
- [x] New homepage copy published in Spanish

## Links to user research, designs and resources

- [Finalized Figma file Desktop](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application))
- [Finalized Figma file Mobile](https://www.figma.com/file/SeSd3LaLd6WkbEYhmtKpO3/Benefits-(Full-Application))

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #1386

## What this PR does
- Model, migration changes: Adds 1 new text fields to `TransitAgency` model, `transit_type`. This field has a `msgid` values.
- Index page change: Use new EN/ES copy
- Agency Index page change: Use new EN/ES copy along with 1 new `TransitAgency` variable

## What this PR does not do
- Add the rest of the new text field variables to `TransitAgency`
- Add new text field variables to `EligibilityVerifier`
- Use the new `TransitAgency` text field variables in any copy aside from the Index, Agency Index pages

## New variable on Transit Agency

Figma, Google | Code in this PR | MST | SacRT
-- | -- | -- | --
[transit type] | `transit_type` | bus | light rail

### Figma
image

### http://localhost:8000/mst
image

### http://localhost:8000/sacrt
image

### CODE REVIEW

File: benefits/core/migrations/0001_initial.py [hunk 1]
```diff
@@ -123,6 +123,12 @@ class Migration(migrations.Migration):
                 ("active", models.BooleanField(default=False)),
                 ("jws_signing_alg", models.TextField()),
                 ("eligibility_index_intro", models.TextField()),
+                ("transit_option", models.TextField()),
+                ("benefit_action", models.TextField()),
+                ("benefit_rate", models.TextField()),
+                ("affected_service", models.TextField()),
+                ("transit_type", models.TextField()),
+                ("card_action", models.TextField()),
```
Comment (machikoyasuda) [inline benefits/core/migrations/0001_initial.py:131]: @thekaveman @angela-tran I'm open to new names for any of these variables. I find `transit_option` and `transit_type` confusing but I wasn't sure what to name it instead, and I'm not sure how they differ.

Comment (machikoyasuda) [reply benefits/core/migrations/0001_initial.py:131]: @thekaveman Should I limit the variables added in this PR to the ones necessary for the Home/Index pages? If so, I'd only need 1 variable and leave the rest for when we actually implement the new copy.

File: benefits/core/migrations/0002_data.py [hunk 2]
```diff
@@ -325,6 +325,12 @@ def load_data(app, *args, **kwargs):
         jws_signing_alg=os.environ.get("MST_AGENCY_JWS_SIGNING_ALG", "RS256"),
         payment_processor=mst_payment_processor,
         eligibility_index_intro=_("eligibility.pages.index.p[0].mst"),
+        transit_option=_("agency.variables.mst.transit_option"),
```
Comment (machikoyasuda) [inline benefits/core/migrations/0002_data.py:328]: @angela-tran @thekaveman Also open to suggestions on how to name these `msgIds`: I just made this up `_("agency.variables.mst.transit_option"),`

File: benefits/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -545,27 +582,6 @@ msgctxt "image alt text"
...
-
-msgid "eligibility.pages.start.veteran.required_items[1]"
-msgstr "ID.me"
-
-msgid "eligibility.pages.start.veteran.required_items[2]"
-msgstr "DS Logon"
-
-msgid "eligibility.pages.start.veteran.required_items[3]"
-msgstr "My HealtheVet"
-
```
Comment (machikoyasuda) [inline benefits/locale/es/LC_MESSAGES/django.po:568]: ???? Have no idea why Django did this !!!!!

Comment (machikoyasuda) [reply benefits/locale/es/LC_MESSAGES/django.po]: I know why now 😓 thankfully the changes in #1498 will prevent this clobbering from happening in the future by `bin/makemessages.sh` b/c the `msgid`s will be in the app.

**Review by thekaveman — COMMENTED** [review]
This PR is simpler than #1498 

We can merge this first and I can rebase on this, instead of the other way around.

**Comment by machikoyasuda** [comment]
@thekaveman Rebased

**Review by thekaveman — APPROVED** [review]

**Pull request merged**